.github/workflows/ci_test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        python-version: ['3.7.5']
+        python-version: ['3.7.6']
```

Comment:
Could we also make the commands on nightly/CI the same?
https://github.com/tensorflow/addons/blob/master/.github/workflows/macos_nightly.yml#L11-L29

Reply:
Done